### PR TITLE
[master] Upgrade to latest dependencies

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
```shell
./hack/update-deps.sh --upgrade
./hack/update-codegen.sh
```
/cc slinkydeveloper
/assign slinkydeveloper

### CODE REVIEW

**Review by pierDipi — APPROVED** [review]
/lgtm

**Comment by pierDipi** [comment]
```
=== CONT  TestEventTransformationForTriggerV1Beta1BrokerV1/0
    test_runner.go:158: Couldn't initialize clients: error while retrieving the config-tracing config map: configmaps "config-tracing" not found
        knative.dev/eventing/test/lib.getTracingConfig
        	/home/prow/go/src/knative.dev/eventing-kafka-broker/vendor/knative.dev/eventing/test/lib/client.go:138
        knative.dev/eventing/test/lib.NewClient
        	/home/prow/go/src/knative.dev/eventing-kafka-broker/vendor/knative.dev/eventing/test/lib/client.go:102
        knative.dev/eventing/test/lib.Setup
        	/home/prow/go/src/knative.dev/eventing-kafka-broker/vendor/knative.dev/eventing/test/lib/test_runner.go:152
        knative.dev/eventing/test/e2e/helpers.EventTransformationForTriggerTestHelper
        	/home/prow/go/src/knative.dev/eventing-kafka-broker/vendor/knative.dev/eventing/test/e2e/helpers/broker_event_transformation_test_helper.go:69
        knative.dev/eventing-kafka-broker/test/e2e.runTest.func1
        	/home/prow/go/src/knative.dev/eventing-kafka-broker/test/e2e/broker_event_trasformation_test.go:48
        testing.tRunner
        	/root/.gvm/gos/go1.14.10/src/testing/testing.go:1050
        runtime.goexit
        	/root/.gvm/gos/go1.14.10/src/runtime/asm_amd64.s:1373
    --- FAIL: TestEventTransformationForTriggerV1Beta1BrokerV1/0 (0.20s)
```

**Comment by slinkydeveloper** [comment]
@pierDipi seems like the issue of monitoring, i'm gonna fix it

**Comment by pierDipi** [comment]
Yep, thanks!

**Comment by slinkydeveloper** [comment]
That's weird, the config map is there (I'm looking here https://00f74ba44b977ed75f1f1c180f01ae3cbf82d82e75-apidata.googleusercontent.com/download/storage/v1/b/knative-prow/o/pr-logs%2Fpull%2Fknative-sandbox_eventing-kafka-broker%2F310%2Fpull-knative-sandbox-eventing-kafka-broker-integration-tests%2F1318185319446614016%2Fartifacts%2F9f9b608a-1210-11eb-8ffa-c2dc0fa0ae6d%2Fk8s.dump-e2e-tests.sh.txt?jk=AFshE3XfW9ZKvmKtvFa8WK8TNaF8wkl6DBP3ng4YbtTeDfV9pL8LPmIIbWkTbVXoE6nwVZs1v30izmrSy-1aMMZxNPwI9MhBW5FlWMfVjgoioGGfVhecIlrjDv6SmkONBHJ9H-ezhzFxxEuf4POPzDQO9Bln3n4nJNd9cN61TKpqAZXZbD2Gqaj0Ie6Tz9FllsPer3m-oOjoZ20m6KwDZu8Np8OfuIHfZ6JSBMsB4wDm2OJmmpLNRgo6wXzxYKWIzrn2s82u3GghnHwfr1cmf25_AWLcrDPAiMhAZvk2bDaAh-dMWmETdSvLG_cUQ4b7uIT8mBc-fOsgq5bPmqawCWsAQT5AJB-KkQKTkFDPYR8JBX_MNZEaW_WoGgKLAwYsuNhOSwrTCBuDYYi_RalLhLaWn-279CcWgJ1ttCN5g2pVDTtqRjBbk55QhORy2fLW0gEDFm0rgPR7iNZ8mFIcJD3qYUlGrebZju69iTCQz29f1hsfM0tJe_qwAY5fD_f9Qr7ZquSBxSUN8e3_CniUvRsiW1F5JfBOd8KVE-i8WkXKvlffv3tU--UvPyXE2XDvcAXrVpBL413I5xMJP28h0QPF1I3NiqJH8N0tw4y6W3e15k6ZZkGad9qnm6BzG8DMbVBW8a1HY0OKHPY1RReY8qopboL0eNVrnHsl-QHGo6lM9_wsK2vfst4wpECLtvyjmz2SxSnKDe94aoROhGmljdMsiOwKLOrK67XBdiNMpSoMn6Xq5LKgcjBH6bWIjZbWrw0YPfw9lOSsO5ntd9Njcx66pzLmwsFcWYA0vxFbRReBZRhXAaqW-ZigkdL58UGUa2gz-7w7GFfKGrjvV4bZIGxmHLsGel-vUbglR90bkQ1MEFDuqfRLVtaYm0m86qfPAFykiSxzS4sZwdekWHNDfR5cclNyloXQ1YKYPCV04nIHpoWBQh-Lqe-hFnrJP8MDIzuQbYd3PeiSkcVOjL0kIEE6PVTuPbKPgBjMHX2ZkM2iU8NLAKkYI-mXTRFPrOrglKvp90g4NyJxdE-1tJKHpph5yPWM-fu1vHaEYKxWqPN5eowZupFHXBrJTNQCVLOjmpZwIxQErioE_1sXrrXymsTAHVo0ZuQVtrOGuK_Vvk0h061uLCRJ963LAT5SpewIpD8W6X-brL5OMg&isca=1)

File: vendor/knative.dev/eventing/test/lib/client.go [hunk 1]
```diff
@@ -133,7 +133,7 @@ func (c *Client) runCleanup() (err error) {
 }
 
 func getTracingConfig(c kubernetes.Interface) (corev1.EnvVar, error) {
-	cm, err := c.CoreV1().ConfigMaps(resources.SystemNamespace).Get(context.Background(), configtracing.ConfigName, metav1.GetOptions{})
+	cm, err := c.CoreV1().ConfigMaps(system.Namespace()).Get(context.Background(), configtracing.ConfigName, metav1.GetOptions{})
```
Comment (slinkydeveloper) [inline vendor/knative.dev/eventing/test/lib/client.go:136]: This seems the issue

Comment (slinkydeveloper) [reply vendor/knative.dev/eventing/test/lib/client.go:136]: here it is we need to fix this one https://github.com/knative/eventing/commit/7eb6009b0c99ac75e2eac0beac31b72ace34ccf1#diff-fbb940d8ac3d40d6c801e8a40575b2a3cd6d6e3f1d2d8f8122fbc2c438a6dccfR67

**Pull request closed without merging**